doc/getting-started/install.md

```diff
@@ -50,23 +50,23 @@ If you are using a different flavor of Linux, you will need to use the correct p
     mkdir -p ~/.local/bin
     mv cabal ~/.local/bin/
 
-Verify that ~/.local/bin is in your PATH
+Verify that ~/.local/bin is in your PATH:
 
     echo $PATH
 
 If `~/.local/bin` is not in the PATH, you need to add the following line to  your `.bashrc` file
 
     export PATH="~/.local/bin:$PATH"
 
-and source the file
+and source the file:
 
     source .bashrc
 
-Update cabal
+Update cabal:
 
     cabal update
 
-Confirm that you installed cabal version `3.2.0.0`.
+Confirm that you installed cabal version `3.2.0.0`:
```

Comment:
I reckon we can use `3.4.0.0` now since it has been released and we're using it in GH.